concordium-consensus/tests/consensus/ConcordiumTests/Afgjort/WMVBA.hs

```diff
@@ -47,8 +47,8 @@ genBlsKeys n = do
   vectorOf n genBlsKeyPair
 
 -- TODO: generate random string
-genByteString :: Gen BS.ByteString
-genByteString = BS.pack <$> vector 8
+genByteString' :: Gen BS.ByteString
+genByteString' = BS.pack <$> vector 8
```

Comment:
Why? 

This is not a good name. What does the `'` signify?

Reply:
Again a compiler error due to name resolution. I put the `'` to resolve the issue. Any better idea?